examples/search_examples/public/search/app.tsx

```diff
@@ -304,14 +317,30 @@ export const SearchExamplesApp = ({
 
       setRequest(searchSource.getSearchRequestBody());
       const abortController = new AbortController();
+      const inspector: IInspectorInfo = {
+        adapter: new RequestAdapter({ handleWarnings: data.search.showWarnings }),
```

Comment:
`RequestAdapter` has a new options object (optional) to give it the ability to show warnings from all the responses in its collection of requests.

Reply:
user doesn't always have control over adapter creation so I think we shouldn't add ay additional options to the adapters constructors.

Reply:
resolved - there are no longer changes to inspector RequestAdapter in this PR